js/connection.js

```diff
@@ -83,15 +88,15 @@
 			});
 		},
 		joinRoom: function(token) {
-			if (signaling.currentRoomToken === token) {
+			if (this.app.signaling.currentRoomToken === token) {
 				return;
 			}
 
 			OCA.SpreedMe.webrtc.leaveRoom();
```

Comment:
Next step is to call signaling here and make webrtc listen to an event from it, so that webrtc subscribes to webrtc instead of signaling being invoked by webrtc.

However I don't get the _trigger, emit and on of signaling. Is that some event handling, or just custom code? @fancycode

Reply:
This is used by SimpleWebRTC for event handling.

Reply:
Okay, so we need a new similar handling on the signaling class, thanks